setup.py

```diff
@@ -16,8 +16,8 @@
 # By subclassing build_extensions we have the actual compiler that will be used which is really known only after finalize_options
 # http://stackoverflow.com/questions/724664/python-distutils-how-to-get-a-compiler-that-is-going-to-be-used
 compile_options = {
-    "msvc": ["/Ox", "/EHsc"],
-    "other": ["-O3", "-Wno-strict-prototypes", "-Wno-unused-function"],
+    "msvc": ["/Ox", "/EHsc", "/std:c++14"],
```

Comment:
Any reason we're going for C++14 over C++11 (like in in [spaCy](https://github.com/explosion/spaCy/blob/b3093367125294d57c2d34593f8dfb06b32de9f0/setup.py#L228))? Not that I mind; in fact, I think we can now tick the standard to C++17 across all our repos - `gcc`, `clang` and `msvc`have had full support for years now.

Reply:
`make_unique` is not available in C++11